### PR TITLE
Added braking engaged symbol

### PR DESCRIPTION
**Pull request by BauerJustin**
Braking engaged symbol with demo code.

### CODE REVIEW

**Comment by davendramaharaj1** [comment]
Task linked: [CU-jx26w1 Add brake engaged](https://app.clickup.com/t/jx26w1)

File: software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.ts [hunk 1]
```diff
@@ -18,7 +19,17 @@ export class SpeedComponent implements OnInit {
   updateSpeed() {
     // Gauge updates every 50ms
     // Graph updates every 5s
-    this.currSpeed += 0.1 * (Math.round(Math.random()) * 4 - 1);
+
+    var i = 0.1 * (Math.round(Math.random()) * 4 - 1)
```
Comment (MJavaadAkhtar) [inline software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.ts:23]: Let's use a more meaningful variable name.

Comment (BauerJustin) [reply software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.ts]: Fixed

File: software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.html [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+        right: 0;
+        top: 35%;
+        text-align: right;
+        width: 20%;
+        font-size: 0.50rem;
+        color: red;
+    }
+</style>
 <div style="text-align: center;">
+    <div [hidden]="!engaged" class="text">
```
Comment (MJavaadAkhtar) [inline software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.html:16]: I am not sure if we should just go with class name "text" (compiled Angular might scream at us). Maybe add a another class name, maybe "brake" and in css call ".text.brake"

Comment (BauerJustin) [reply software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.html]: Sorry, I copied in pasted the CSS and did not change the name. Fixed now.

**Review by ArnavJain23 — APPROVED** [review]

**Pull request merged**